default/AI/ShipDesignAI.py

```diff
@@ -77,8 +77,12 @@
 MISSING_REQUIREMENT_MULTIPLIER = -1000
 INVALID_DESIGN_RATING = -999  # this needs to be negative but greater than MISSING_REQUIREMENT_MULTIPLIER
 
+
+WITH_UPKEEP = 'with upkeep'
+WITHOUT_UPKEEP = 'without outkeep'
```

Comment:
Spelling. Also "(not) considering fleet upkeep" is more true to the meaning I think.